masonry/src/widget/widget.rs

```diff
@@ -126,7 +126,7 @@ pub trait Widget: AsAny {
 
     fn accessibility_role(&self) -> Role;
 
-    fn accessibility(&mut self, ctx: &mut AccessCtx);
+    fn accessibility(&mut self, ctx: &mut AccessCtx, builder: &mut NodeBuilder);
```

Comment:
Reading through the rest of this code, my preference would be for this variable to called `node` instead of the extremely generic `builder`.

However, if the idiomatic variable name used for this type in accesskit is `builder`, then I guess that's fine.

Reply:
`builder` has become the idiomatic variable name by default. I've used it in egui and in AccessKit's own tests and examples. So far, no one else has pushed back on that. But I'm willing to reconsider it.